### PR TITLE
Remove preupg-create-group-xml utility

### PR DESCRIPTION
**Pull request by bocekm**
There's no need to be generating the all-xccdf.xml (descriptor for OpenSCAP) from a single module by using _preupg-create-group-xml_. _preupg-xccdf-compose_ can be used for exactly that.

### CODE REVIEW

File: preupg/xmlgen/compose.py [hunk 1]
```diff
@@ -331,20 +315,19 @@ def get_xml_tree():
...
 
     @staticmethod
-    def run_compose(dir_name, content=None, generate_from_ini=True):
+    def run_compose(dir_name, generate_from_ini=True):
         target_tree = ComposeXML.get_xml_tree()
         settings.UPGRADE_PATH = dir_name
         if os.path.exists(os.path.join(dir_name, settings.file_list_rules)):
             os.unlink(os.path.join(dir_name, settings.file_list_rules))
-        group_xmls = ComposeXML.collect_group_xmls(dir_name, content, 0,
+(??)        group_xmls = ComposeXML.collect_group_xmls(dir_name, 0,
```
Comment (AloisMahdal) [inline preupg/xmlgen/compose.py]: `(??)`??

Comment (bocekm) [inline preupg/xmlgen/compose.py]: Fixed.

Comment (bocekm) [inline preupg/xmlgen/compose.py]: Actually I most probably fixed it before you did the review. Maybe you had it opened for a few hours and then proceeded with the review :)

**Comment by bocekm** [comment]
@MajoDurco message rewritten.

**Comment by pirat89** [comment]
--test--

**Comment by pirat89** [comment]
--test--

**Comment by pirat89** [comment]
--test--

**Comment by bocekm** [comment]
@MajoDurco - message rewritten, you can check it and finish the review. Thanks.

File: preupg/creator/settings.py [hunk 2]
```diff
@@ -43,7 +43,7 @@
 properties_ini = "- Fill empty options in %s before you use it."
 prop_src_version = "Specify major source OS version e.g. \"6\":"
 prop_dst_version = "Specify major destination OS version e.g. \"7\":"
-
-text_for_testing = "\nFor testing content run these two commands:\n" \
-                   "- preupg-create-group-xml %s to create XML file\n" \
-                   "- preupg -c %s"
```
Comment (MajoDurco) [inline preupg/creator/settings.py]: now result message is:
`preupg -c modules-results/system/packages/all-xccdf.xml`
but needs to be:
`preupg -c modules-results/all-xccdf.xml`

Comment (bocekm) [reply preupg/creator/settings.py]: Thanks. Fixed.

File: preupg/xmlgen/compose.py [hunk 3]
```diff
@@ -80,8 +80,7 @@ def get_compose_dir_name(self):
 class ComposeXML(object):
 
     @staticmethod
-    def collect_group_xmls(module_set_dir, source_dir, content=None,
-                           generate_from_ini=True):
+    def collect_group_xmls(module_set_dir, source_dir, generate_from_ini=True):
         """
         Find group.xml file recursively through all module directories
         and modules. Collect data from each of them into dictionary.
```
[the remark this answers is not in the document]
Comment (bocekm) [reply preupg/xmlgen/compose.py]: Removed.

**Comment by AloisMahdal** [comment]
qa_ack: Just verify the binary is gone.

This is also covered by downstream test now.

**Comment by AloisMahdal** [comment]
Verified with preupgrade-assistant-2.4.4-0.201708291626Z.PR289.el6_8

**Pull request merged**